README.md

```diff
@@ -3,561 +3,34 @@
-Creates a skipped unit test case.  It will never execute.
-
-Parameters:
-* `description as string` - a string describing the test case executed by `func`
-* `func as object` - the (skipped) function to execute as part of this test case
-* `args = invalid as dynamic` - optional parameter that will be passed in `func` as an argument
-
-Example:
-```brightscript
-m.xit("a omitted test case", sub()
-    m.fail() ' this test will never be executed, so it doesn't matter if it's explicitly failed
-end sub)
-```
-
-#### `m.it_each(arrayOfArgs as object, descriptionGenerator as object, func as object)`
-Creates a parameterized test cases with generated description.
-Will be expanded as `m.it` in for cycle with the passing argument to `func` from arguments array
-
-Parameters:
-* `arrayOfArgs as object` - the array of args that will be passed in `func`
-* `descriptionGenerator as object` - a function that generate description string
-* `func as object` - the array of args, these args will be passed in `func`
-
-Example:
-```
-m.it_each([
-    [1, 1],
-    [2, 1],
-],
-function (args)
-    return substitute("compare {0} with {1}", args[0].tostr(), args[1].tostr())
-end function,
-sub(args)
-    if args[0] = args[1] then m.pass() else m.fail()
-end sub)
-```
-
-#### `m.fit_each(arrayOfArgs as object, descriptionGenerator as object, func as object)`
-Creates a parameterized test cases with generated description.
-Will be expanded as `m.fit` in for cycle with the passing argument to `func` from arguments array
-
-Parameters:
-* `arrayOfArgs as object` - the array of args that will be passed in `func`
-* `descriptionGenerator as object` - a function that generate description string
-* `func as object` - the array of args, these args will be passed in `func`
-
-Example:
-```
-m.fit_each([
-    [1, 1],
-    [2, 1],
-],
-function (args)
-    return substitute("focused test, compare {0} with {1}", args[0].tostr(), args[1].tostr())
-end function,
-sub(args)
-    if args[0] = args[1] then m.pass() else m.fail()
-end sub)
-```
+## Contributing
 
-#### `m.xit_each(arrayOfArgs as object, descriptionGenerator as object, func as object)`
-Creates a parameterized test cases with generated description.
-Will be expanded as `m.xit` in for cycle with the passing argument to `func` from arguments array
+If you spot a bug or want a feature, please feel free to file an issue. If you want to contribute code yourself, simply fork the repo and open a PR.
 
-Parameters:
-* `arrayOfArgs as object` - the array of args that will be passed in `func`
-* `descriptionGenerator as object` - a function that generate description string
-* `func as object` - the array of args, these args will be passed in `func`
+### Modifying the docs site
 
-Example:
+We use [docsify](https://docsify.js.org) to build our site, and [Github pages](https://pages.github.com/) to deploy it. To run the documentation site locally, first install docsify:
+```shell
+$ npm i docsify-cli -g
 ```
```

Comment:
Think it'd be worth taking `docsify-cli` as a `devDependency`?  That'd make this as simple as `npm start` or `npm serve` or something 😃